frontend/lang/messages/en-US.json

```
@@ -1193,9 +1193,9 @@
     "notifiers": "Notifiers",
     "notifiers-description": "Set up email and push notifications that trigger on specific events.",
     "manage-data": "Manage Data",
-    "manage-data-description": "Manage your Food and Units (more options coming soon)",
+    "manage-data-description": "Manage your Mealie data; foods, units, categories, tags and more.",
```

I think at least Foods after the semicolon should be capitalized and for consistency i would capitalize Units, Categories and Tags as well. 
```suggestion
    "manage-data-description": "Manage your Mealie data; Foods, Units, Categories, Tags and more.",
```